cdap-ui/app/tracker/controllers/preview-ctrl.js

```diff
@@ -38,19 +39,22 @@ class TrackerPreviewController {
       scope: this.$scope
     };
 
-    let entityType;
     if (this.$state.params.entityType === 'streams') {
-      entityType = 'stream';
+      this.entityType = 'stream';
     } else if (this.$state.params.entityType === 'datasets') {
-      entityType = 'dataset';
+      this.entityType = 'dataset';
     }
 
-    let query = 'SELECT * FROM ' + entityType + '_' + this.$state.params.entityId + ' LIMIT 500';
+    let query = 'SELECT * FROM ' + this.entityType + '_' + this.$state.params.entityId + ' LIMIT 500';
```

Comment:
This query is incorrect. We should use the `ExploreTablesStore` to get the database name and table name. Please refer Explore Fast action in the react code base